.buildkite/pipelines/security_solution/api_integration.yml

```diff
@@ -219,6 +219,17 @@ steps:
         - exit_status: '1'
           limit: 2
 
+  - label: Running rule_delete:essentials:qa:serverless
+    command: .buildkite/scripts/pipelines/security_solution_quality_gate/api-integration-tests.sh rule_delete:essentials:qa:serverless
+    key: rule_delete:essentials:qa:serverless
+    agents:
+      queue: n2-4-spot
+    timeout_in_minutes: 120
+    retry:
+      automatic:
+        - exit_status: '1'
+          limit: 2
```

Comment:
Another thing I'd do differently is the grouping of tests under `x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management`. I feel like `rule_delete` etc are probably too fine-grained groups, and I'd try combining them into something like `rules_crud`, `rules_bulk_crud`, `rules_bulk_actions`, etc.

But this is out of the scope of this PR, just sharing ideas.

Reply:
I like that! How we group them also has to do with how long the tests take to execute, so sometimes breaking them out into finer grained groups puts them under the threshold for time to execute. But we could still at least group them within the folders together even if we then need to chunk the configs.

Reply:
Yep, agreed!